apps/frontend/src/app/legal/legal/legal.component.html

```diff
@@ -89,5 +89,19 @@ <h3>User Rights</h3>
       </p>
     </li>
   </ul>
-  <h3>Contact Us</h3>
+  <div *ngIf="imprint">
```

Comment:
I wonder if this even makes sense, the imprint object will be available every time, but the properties may be undefined or empty.

Reply:
Yeah you are right. It would probably only show if the GET-call fails completly.

Reply:
Yeah, we can keep this one. It makes sure that we don't have to use the `?`-operator + if the worst-case happens and the `GET`-request fails.